java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/OrderController.java

```diff
@@ -27,10 +27,10 @@ public List<FoodTruck2> menuAppetizer(){
     }
 
     //------------LIST ONLY OF ENTREE-----------//
-    @GetMapping("/entree")
-    public List<EntreePlate> menuEntree(){
-        return entreeService.menu();
-    }
+//    @GetMapping("/entree")
+//    public List<EntreePlate> menuEntree(){
+//        return entreeService.menu();
+//    }
```

Comment:
commented out code is a no-no, right? Should this be removed?

Reply:
Sure, I forgot about this